templates/gsl.install.sh

```diff
@@ -166,10 +166,16 @@ build_from_tarball()
     # Join generated and command line options.
     local CONFIGURATION=("${OPTIONS[@]}" "$@")
 
-    configure_options "${CONFIGURATION[@]}"
-    make_jobs $JOBS --silent
-    make install
-    configure_links
+    if [[ $ARCHIVE == $MBEDTLS_ARCHIVE ]]; then
+        sed -i "s~DESTDIR=/usr/local~DESTDIR=${PREFIX}~" Makefile
+        make -j $JOBS lib
+        make install
+    else
```

Comment:
@pmienk determined the sed line isn't necessary.  The other make lines I believe are though.

Reply:
The Makefile hardcoded the destdir, which obviously doesn't respect our prefix dir.  The sed line was to replace it dynamically with the user specified destdir.  I haven't gotten to all of the feedback from @pmienk yet, but he indicated you can just run `DESTDIR=/foo/bar make ...` to have the same effect.

Reply:
it doesn't use autotools, likely because of the number of embedded scenarios it's designed to run on.  It comes with a Makefile and it also looks like it's cmake enabled.

Reply:
Ok, st a minimum a comment would be good there. But I think this can also be cleaned up with a generated intermediate build file so that the change is retained within the dependency code and not a condition in the general build step.